src/meta/src/hummock/manager/mod.rs

```diff
@@ -2738,7 +2738,16 @@ async fn write_exclusive_cluster_id(
 
     let cluster_id_dir = format!("{}/{}/", state_store_dir, CLUSTER_ID_DIR);
     let cluster_id_full_path = format!("{}{}", cluster_id_dir, CLUSTER_ID_NAME);
-    let metadata = object_store.list(&cluster_id_dir).await?;
+    let metadata = match object_store.list(&cluster_id_dir).await {
+        Ok(metadata) => metadata,
+        Err(_) => {
+            return Err(ObjectError::internal(
+                "Fail to access remote object storage, 
+            please check if your Access Key and Secret Key are configured correctly. ",
+            )
+            .into())
+        }
```

Comment:
This failure can happen for other reasons. Shall we just match S3's `DispatchError`?

Reply:
The high probability of the error occurs here is that the configuration is incorrect, I just want to remind the user to check ak/sk, and make it simple. And the object storage is not only s3.